src/app/modules/examples/examples.utils.ts

```diff
@@ -0,0 +1,224 @@
+import { StatusCodes } from 'http-status-codes'
+import moment from 'moment-timezone'
+
+import { createLoggerWithLabel } from '../../../config/logger'
+import { MapRouteError } from '../../../types/routing'
+import { DatabaseError } from '../core/core.errors'
+
+import { ResultsNotFoundError } from './examples.errors'
+import {
+  addAvgFeedback,
+  filterByAgencyId,
+  filterBySubmissionCount,
+  filterInactiveAndUnlistedForms,
+  groupSubmissionsByFormId,
+  lookupAgencyInfo,
+  lookupFormFeedback,
+  lookupFormInfo,
+  projectAvgFeedback,
+  projectFormDetails,
+  projectSubmissionInfo,
+  searchFormsById,
+  searchFormsWithText,
+  searchSubmissionsForForm,
+  sortByCreated,
+  sortByLastSubmitted,
+  sortByRelevance,
+} from './examples.queries'
+
+const logger = createLoggerWithLabel(module)
+
+/**
+ * Handler to map ApplicationErrors to their correct status code and error
+ * messages.
+ * @param error The error to retrieve the status codes and error messages
+ * @param coreErrorMessage Any error message to return instead of the default core error message, if any
+ */
+export const mapRouteError: MapRouteError = (error) => {
```

Comment:
would this not be better placed in `examples.errors.ts`?

Reply:
Isn't this an util? errors should only hold the errors

Reply:
alright, thinking again, you make sense